get-started/using/index.md

```diff
@@ -30,23 +30,23 @@ title: Using Monero
                 <div class="info-block">
                     <div class="row center-xs">
                         <div class="col">
-                            <h2>2. Support</h2>
+                            <h2>Request Support</h2>
                         </div>
                     </div>
                     <div class="row start-xs">
-                        <p>Before reading further, it is important to know that there is a large, supportive community that can assist if your run into trouble. If you have questions about anything, no matter how large or small, don't hesitate to ask. We chill in a lot of places, so see our <a href="/community/hangouts/">Hangouts page</a> to track us down. Remember, there are no stupid questions, and we are all learning together.</p>
+                        <p>There is a large and supportive community that will assist if you experience any difficulty. See our <a href="/community/hangouts/">Hangouts page</a> for more information.</p>
                     </div>
                 </div>
             </div>
             <div class="right half col-lg-6 col-md-6 col-sm-12 col-xs-12">
                 <div class="info-block">
                     <div class="row center-xs">
                         <div class="col">
-                            <h2>3. Getting a Wallet</h2>
+                            <h2>Obtain a Wallet</h2>
                         </div>
                     </div>
                     <div class="row start-xs">
-                        <p>You need somewhere to store physical cash such as your bank, your wallet, or under your mattress. Although Monero is an entirely digital asset (you cannot touch them!), you still need a place to store them. This is done in a digital wallet. You oversee securing your own funds, and there is no way to restore them if you make a mistake. Thus, take great care in selecting the right wallet for you. Some are safer and others are more convenient than others. See our <a href="/downloads/">downloads page</a> for the wallets that The Monero Project has put out.</p>
+                        <p>A Monero wallet is required to secure your own funds. See the <a href="/downloads/">downloads page</a> for a listing of available wallets.</p>
```

Comment:
If we're going to use `Obtain Monero` for the next section I think we should probably pick another verb here.  (`Getting` is fine IMO and `Obtain` sounds a bit stodgy in this context.)

`your own` is redundant. If they're mine (your), then who else would they belong to.

Honestly, I think also think we need to disambiguate from the beginning the difference between a wallet (the software) and a wallet (an account). I think `account` is the preferred term with the developers for this very reason. Maybe:
 
> <p>A Monero wallet is the software that's used secure your funds. See the <a href="/downloads/">downloads page</a> for a listing of available wallets. This will allow you to establish an account that only you control.</p>